dashboard/app/controllers/learning_goal_teacher_evaluations_controller.rb

```diff
@@ -0,0 +1,60 @@
+class LearningGoalTeacherEvaluationsController < ApplicationController
+  authorize_resource
```

Comment:
nonblocking: I think you could use `load_and_authorize_resource` here, which can be a nice way to make sure that the resource you are loading is exactly what you authorized. these are the docs I am looking at, which make it look like they would do exactly what you are already doing, at least for create and update: https://github.com/CanCanCommunity/cancancan/blob/develop/docs/controller_helpers.md#authorize_resource-load_resource-load_and_authorize_resource